docs/development/extensions-core/stats.md

```diff
@@ -125,8 +126,24 @@ To acquire standard deviation from variance, user can use "stddev" post aggregat
 }
 ```
 
+#### Druid SQL
```

Comment:
It would be preferable to move the SQL query section before the native query section since SQL is more widely used and familiar to users. Same suggestion for all the SQL sections in this page.